scripts/harmonize.py

```diff
@@ -236,7 +241,8 @@ def run():
               beta_col = args.beta_col,
               require_gnomad = args.require_gnomad,
               passing_only = args.passing_only,
-              gnomad_min_an = args.gnomad_min_an)
+              gnomad_min_an = args.gnomad_min_an,
+              gnomad_max_abs_diff = args.gnomad_max_abs_diff)
```

Comment:
I think we should have a filter for very big differences by default in the conf... 
like 0.1 or 0.2 if they just are not the same variant. We could look at the plots if there are reasonable cutoffs

Reply:
I suggest we add that later once we have come to an agreement on what would be a reasonable default. I'll draw some plots and fetch some numbers for different cutoffs.

Reply:
yea sounds good but having really big difference is always safe.... over .2

Reply:
True!